modules/oe_theme_content_call_proposals/templates/oe_theme_content_call_proposals_publication_details.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+/**
+ * @file
+ * Theme for the publication details extra field on Call for proposals content type.
+ *
+ * @see \Drupal\oe_theme_content_call_proposals\Plugin\ExtraField\Display\CallForProposalsPublicationDetailsExtraField::viewElements()
+ */
+#}
+{{ date }}
+{% if link is not empty %}
+  {{ 'in'|t }}
```

Comment:
It's quite tricky to use ecl-twig/link and avoid escaping using t() so it is implemented so.